pkg/manager/member/scaler.go

```diff
@@ -104,3 +113,45 @@ func ordinalPVCName(memberType v1alpha1.MemberType, setName string, ordinal int3
 func ordinalPodName(memberType v1alpha1.MemberType, tcName string, ordinal int32) string {
 	return fmt.Sprintf("%s-%s-%d", tcName, memberType, ordinal)
 }
+
+// scaleOne calculates desired replicas and delete slots from actual/desired
+// stateful sets by allowing only one pod to be deleted or created
+// it returns following values:
+// - scaling:
+//   - 0: no scaling required
+//   - 1: scaling out
+//   - -1: scaling in
```

Comment:
Could we define enums for this?

Reply:
I don't think it's necessary because it's used internally and I actually uses `> 0` and `< 0` to decide whether to scale in or scale out.